### PR TITLE
Ch boost tests 167334033

### PR DESCRIPTION
**Pull request by hezronkimutai**
#### What does this PR do?
This PR increases test coverage
#### Description of Task to be completed
This PR's main aim was to increase test coverage.

#### How should this be manually tested?
- Git clone this REPO.
- git checkout ch-boost-tests-167334033 branch.

#### What are the relevant pivotal tracker stories?
[#167334033](https://www.pivotaltracker.com/n/projects/2353827/stories/167334033)

### CODE REVIEW

File: api/v2/models/queries.js [hunk 1]
```diff
@@ -8,10 +8,12 @@ const createTableUsers = `CREATE TABLE IF NOT EXISTS users(
     address VARCHAR NOT NULL,
     isadmin BOOLEAN NOT NULL,
     password VARCHAR NOT NULL,
-    profilePic VARCHAR NULL
+    profilePic VARCHAR NULL,
+    createdon timestamp default now()
     )`;
 const createTableProperties = `CREATE TABLE IF NOT EXISTS properties(
       id serial PRIMARY KEY,
+      owner INTEGER,
```
Comment (TheoOkafor) [inline api/v2/models/queries.js]: Make foreign key reference here to the user table

Comment (TheoOkafor) [reply api/v2/models/queries.js]: make it cascade on delete.

**Review by TheoOkafor — APPROVED** [review]

**Review by TheoOkafor — CHANGES_REQUESTED** [review]
Attend to the comments

File: api/v2/models/queries.js (hunk 1 above)
Comment (hezronkimutai) [reply api/v2/models/queries.js]: I received the feedback and already worked on it.

**Pull request merged**